ci/prepare-unity.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
in general: no way to get this into a c# script?

Reply:
It will be much more work for a small script that is only run by CI

Reply:
I think it may be a good idea to (at some point) separate out scripts that are meant for use by users (codegen, build-client, etc.) and scripts that will only be run by ci (cc @samiwh).